### ISSUE TITLE
Specific statement is incorrect

### ISSUE THREAD

**Issue by kannanwisen** [angular/angular#46424]
### Description

According the documentation:

` "Additionally, Angular applies the ng-submitted class to <form> elements upon submission. This class does not apply to inner controls."`

According to the ngControlStatusHost const and AbstractControlStatus class, there is no ng-submitted option. 

Kindly remove this statement. 

Thanks

Kannan.S

### What is the affected URL?

https://angular.io/guide/forms

### Please provide the expected behavior vs the actual behavior you encountered

Kindly remove this statement.

**Comment by jnizet** [comment]
Hi @kannanwisen 

Here's a demo showing that the documentation is correct: when you submit a form, an `ng-submitted` CSS class is added to the form element. But not to its controls. https://stackblitz.com/edit/angular-ivy-as7vyy?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.css

**Comment by kannanwisen** [comment]
Still I feel it is misleading. You have used ngSubmit. I have implemented HTML submit. I feel the documentation should say that it is Angular ngSubmit

**Comment by jnizet** [comment]
Well, the guide is about Angular forms, so I think it's implied. But well, the maintainers will decide what to do.

**Comment by bob-watson** [comment]
How about rewording that section to read something like:

## Track form states

Angular applies the `ng-submitted` class to `<form>` elements after the form has been submitted. This class can be used to change the form's style after it has been submitted.

## Track control states

Adding the [NgModel](https://angular.io/api/forms/NgModel) directive to a control adds class names to the control that describe its state. These classes can be used to change a control's style based on its state.

The following table describes the class names that Angular applies based on the control's state.

(table goes here)
